tests/transformers/test_multi_head_attention.py

```
@@ -0,0 +1,93 @@
+            .reshape(batch_size, n_heads, seq_len, d_model)
+        )
+        query, key, value = x, x, x
+        mask = (torch.triu(x, diagonal=1) == 0).to(device)
+        attn_values, attn_weights = attention(query=query, key=key, value=value, mask=mask, dropout=dropout)
+
+        # Get Mask indices from mask
+        non_mask_indices = torch.nonzero(mask, as_tuple=True)
+        assert (
+            torch.all(attn_values[non_mask_indices] == -1e9),
+            "Attention values should be -1e9 for masked indices",
+        )
```

Ruff (0.8.2)

Assert test is a non-empty tuple, which is always `True`

(F631)